ci/infra/testrunner/tests/conftest.py

```diff
@@ -1,9 +1,20 @@
+import contextlib
+import copy
+import logging
+import os
 import pytest
 import platforms
+import tempfile
+import shutil
 from skuba import Skuba
 from kubectl import Kubectl
 from utils import BaseConfig
 
+
+LOGGER = logging.getLogger('conftest')
```

Comment:
This will make test logs go to a different logger than the testrunner's log. Not sure this is convenient.